desktop/release-notes.md

```diff
@@ -24,11 +24,56 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## 4.20.0
+
+{% include release-date.html date="2023-05-25" %}
+
+{% include desktop-install.md all=true version="4.20.0" build_path="/" %}
```

Comment:
To-do: Update the checksums 
(Although I know this is already on your radar as it can only be done last minute)

Reply:
@lorenrh are these ready to be added yet?